packages/cli/lib/prompts/previewPrompt.js

```diff
@@ -34,6 +34,24 @@ const previewPrompt = (promptOptions = {}) => {
   ]);
 };
 
+const previewProjectPrompt = async themeComponents => {
```

Comment:
Would you mind updating this file's name to be a little more specific? Maybe something along the lines of `themePreviewPrompt`?